db/schema.rb

```diff
@@ -161,17 +161,17 @@
 
   create_table "other_assets_declarations", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.uuid "legal_aid_application_id", null: false
-    t.decimal "second_home_value"
-    t.decimal "second_home_mortgage"
-    t.decimal "second_home_percentage"
-    t.decimal "timeshare_value"
-    t.decimal "land_value"
-    t.decimal "jewellery_value"
-    t.decimal "vehicle_value"
-    t.decimal "classic_car_value"
-    t.decimal "money_assets_value"
-    t.decimal "money_owed_value"
-    t.decimal "trust_value"
+    t.decimal "second_home_value", precision: 14, scale: 2
+    t.decimal "second_home_mortgage", precision: 14, scale: 2
+    t.decimal "second_home_percentage", precision: 14, scale: 2
+    t.decimal "timeshare_value", precision: 14, scale: 2
+    t.decimal "land_value", precision: 14, scale: 2
+    t.decimal "jewellery_value", precision: 14, scale: 2
+    t.decimal "vehicle_value", precision: 14, scale: 2
+    t.decimal "classic_car_value", precision: 14, scale: 2
+    t.decimal "money_assets_value", precision: 14, scale: 2
+    t.decimal "money_owed_value", precision: 14, scale: 2
+    t.decimal "trust_value", precision: 14, scale: 2
```

Comment:
The `, precision: 14, scale: 2` entries I think are an artifact of a previous bit of work you did where you were setting these values. The old version of `schema.rb` is the correct one

Reply:
This is fixed now.